### ISSUE TITLE
How to build and install?

### ISSUE THREAD

**Issue by h7io** [wolfgangimig/itol#63]
After playing around with gradle and the dependencies I managed to get a `itol-addin-1.0.jar`. It does not run with `java -cp itol-addin-1.0.jar com.wilutions.itol.IssueApplication` though.

Can you add instructions on how to build and then how to install the built add-on? 

Thank you

**Comment by wolfgangimig** [comment]
Yes, it is currently difficult to build ITOL without additional information. 
I have added a short manual here: https://github.com/wolfgangimig/itol/wiki/How-to-Build. 
Hope this helps.

**Comment by xomix** [comment]
I have followed your instructions and after running ant on deploy.xml I get an msi in bundle dir. When installing the .msi I get an error: "Class com/wilutions/redmineaddin/IssueApplication not found." And then "Failed to launch JVM". Looking in the bundle/dist/32/Issue Tracker for Microsoft Outlook 32bit.jar the class is there...
Can you help me with this?
Thanks in advance
